api/src/scripts/ticketScript.js

```diff
@@ -0,0 +1,192 @@
+const dataMapper = require("../dataMapper/dataMapper");
+const YoungModel = require("../models/young");
+const ReferentModel = require("../models/referent");
+const TicketModel = require("../models/ticket");
+const TagModel = require("../models/tag");
+const { formatDistance } = require('date-fns');
+const { fr } = require('date-fns/locale');
+
+(async () => {
+  console.log("AM I HERE ???");
+  // Mes étapes
+  // 1. Récupérer les différentes tables et les filtrer pour récupérer les champs intéressants
+  const tickets = await dataMapper.getAllTickets();
+  const ticketArticles = await dataMapper.getAllTicketArticles();
+  const ticketStates = await dataMapper.getAllTicketStates();
+  const ticketPriorities = await dataMapper.getAllTicketPriorities();
+  const ticketArticleTypes = await dataMapper.getAllTicketArticleTypes();
+  const ticketArticleSenders = await dataMapper.getAllTicketArticleSenders();
+  const ticketTags = await dataMapper.getAllTags();
+  const tagItems = await dataMapper.getAllTagItems();
+  const groups = await dataMapper.getAllGroups();
+  const users = await dataMapper.getAllUsers();
+
+  function findZammadUser(id) {
+    return users.filter((user) => id === user.id)[0];
+  };
+
+  // 2. Créer / mettre à jour ma table Tag avec toutes les occurences
+  for (let tag of tagItems) {
+    const tagExisting = await TagModel.findOne({ zammadId: tag.id });
+    if (tagExisting) {
+      console.log("TAG ALREADY EXISTS !");
+      if (tagExisting.name !== tag.name) {
+        await tagExisting.save({ name: tag.name });
+      } continue;
```

Comment:
ça marche comme syntaxe ça ? jamais vu !
j'aurais plutôt mis
```suggestion
      } 
      continue;
```

Reply:
@arnaudambro ca fonctionne, c'est comme si je mettais un else { continue } non ? Si je le mets en dessous, ca va `continue` dans tous les cas ?

Reply:
Ou alors pas besoin de le laisser sur la même ligne maybe ? Je suis jamais sûre

Reply:
Je peux re-tester avec les 2 écritures !